src/backend/t/1000-bs_worker.t

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+use Test::More tests => 3;
+
+
+use FindBin;
+use lib "$FindBin::Bin/lib/";
+
+use Test::Mock::BSConfig;
+use Test::Mock::BSRPC;
+
+
+@::ARGV = ('--testcase');
+require_ok('./bs_worker');
+
+$BSConfig::bsdir = "$FindBin::Bin/data/1000";
+
+my $buildinfo = {
+  srcserver  => $BSConfig::srcserver,
+  project    => 'project1',
+  package    => 'package1',
+  srcmd5     => '1234123412341234',
+  verifymd5  => 'f157738ddea737a2b7479996175a6cec',
+};
+
+$Test::Mock::BSRPC::fixtures_map = {
+  "srcserver/getsources?project=project1&package=package1&srcmd5=$buildinfo->{srcmd5}"
+    => 'srcserver/getsources',
+};
+
+my (@got, @expected);
+
+
+# getsources
+
+@got = getsources($buildinfo, $BSConfig::bsdir.'/bs_worker');
```

Comment:
Use $tmpdir instead of $BSConfig::bsdir/bs_worker?

Reply:
Done.